assets/js/blocks/cart-checkout/checkout/inner-blocks/checkout-terms-block/edit.tsx

```diff
@@ -66,41 +68,86 @@ export const Edit = ( {
 				) : (
 					<RichText
 						tagName="span"
-						value={ text || termsConsentDefaultText }
+						value={ currentText }
 						onChange={ ( value ) =>
 							setAttributes( { text: value } )
 						}
 					/>
 				) }
 			</div>
-			{ ! currentText.includes( '<a ' ) && (
+			{ /* Show this notice if a terms page or a privacy page is not setup. */ }
+			{ ( ! TERMS_URL || ! PRIVACY_URL ) && (
 				<Notice
 					className="wc-block-checkout__terms_notice"
 					status="warning"
 					isDismissible={ false }
-					actions={
-						termsConsentDefaultText !== text
-							? [
-									{
-										label: __(
-											'Restore default text',
-											'woo-gutenberg-products-block'
-										),
-										onClick: () =>
-											setAttributes( { text: '' } ),
-									},
-							  ]
-							: []
-					}
+					actions={ [
+						! TERMS_URL && {
+							label: __(
+								'Setup a Terms and Conditions page',
+								'woo-gutenberg-products-block'
+							),
+							onClick: () =>
+								window.open(
+									`${ ADMIN_URL }admin.php?page=wc-settings&tab=advanced`,
+									'_blank'
+								),
+						},
+						! PRIVACY_URL && {
+							label: __(
+								'Setup a Privacy Policy page',
+								'woo-gutenberg-products-block'
+							),
+							onClick: () =>
+								window.open(
+									`${ ADMIN_URL }options-privacy.php`,
+									'_blank'
+								),
+						},
+					].filter( Boolean ) }
 				>
 					<p>
 						{ __(
-							'Ensure you add links to your policy pages in this section.',
+							"You don't seem to have a Terms and Conditions and/or a Privacy Policy pages setup.",
 							'woo-gutenberg-products-block'
 						) }
 					</p>
 				</Notice>
 			) }
+			{ /* Show this notice if we have both a terms and privacy pages, but they're not present in the text. */ }
+			{ TERMS_URL &&
+				PRIVACY_URL &&
+				! (
+					currentText.includes( TERMS_URL ) &&
+					currentText.includes( PRIVACY_URL )
```

Comment:
Instead of just checking if the text contains a link, it would check if it contains both terms and privacy policy links.